### ISSUE TITLE
Prompt Weight (WORDS:1.X) issue with Stable Cascade

### ISSUE THREAD

**Issue by Aamir3d** [comfyanonymous/ComfyUI#2889]
When using Stable Cascade, I found that any prompt that has a weight within the prompt leads to distorted images.
This is using a simple Stable Cascade workflow.

Example Prompt:
_A mushroom with windows and doors,the (chimney is smoking:1.2),it's late in the evening in the forest and the light is shining inside,(Snow White and the seven dwarfs are standing in front of their mushroomhouse:1.4), awaiting the night in the last rays of the sun,( art by John Howe, Kim Keever:1.4),very high details,voluminous light,very high details,voluminous light,hyperrealistic,in autumn,highres,Cinematic Lighting,hoher kontrast , an oil painting, a charcoal drawing, a pastel drawing, Winter, Spring_

![image](https://github.com/comfyanonymous/ComfyUI/assets/122131887/88b1f04c-88ff-40c3-a6bf-3e78afa6829c)

**Example after removing the the (WORDS:1.2) and only including (WORDS)**
A mushroom with windows and doors,the (chimney is smoking),it's late in the evening in the forest and the light is shining inside,(Snow White and the seven dwarfs are standing in front of their mushroomhouse), awaiting the night in the last rays of the sun,(art by John Howe, Kim Keever),very high details,voluminous light,very high details,voluminous light,hyperrealistic,in autumn,highres,Cinematic Lighting,hoher kontrast , an oil painting, a charcoal drawing, a pastel drawing, Winter, Spring

![image](https://github.com/comfyanonymous/ComfyUI/assets/122131887/6297fc5e-4826-4091-839b-2cee4731abe9)

**Comment by Starmann11** [comment]
Yeah i have got the same kinda results. Did you find any fixes for that?
![ComfyUI_00022_](https://github.com/comfyanonymous/ComfyUI/assets/161734086/d8fc6340-d9f5-4717-a215-50d1c17e4e5b)
